config/config.go

```diff
@@ -604,7 +606,7 @@ func GetDefaultConfigPath(workingDir string) string {
 func FindConfigFilesInPath(rootPath string, terragruntOptions *options.TerragruntOptions) ([]string, error) {
 	configFiles := []string{}
 
-	err := filepath.Walk(rootPath, func(path string, info os.FileInfo, err error) error {
+	err := symwalk.Walk(rootPath, func(path string, info os.FileInfo, err error) error {
```

Comment:
I was wondering why `filepath.Walk()` wasn't replaced in other places, for example in `terraform/source.go`, I think it may lead to the failure of detection of changes in modules

https://github.com/search?q=repo%3Agruntwork-io%2Fterragrunt%20filepath.Walk&type=code

Reply:
I made the change.